### PR TITLE
[CI] Update CI matrix in Travis

### PR DESCRIPTION
**Pull request by SilleBille**
Remove EOL release and add latest

**Blocker:** Currently blocked until the PR gets merged in freeipa: https://github.com/freeipa/freeipa/pull/3783

**Update:** Since the blocker is pending review, this patch moves running IPA tests on F31, from "Mandatory" tests to "Optional" tests
 
`Signed-off-by: Dinesh Prasanth M K <dmoluguw@redhat.com>`

### CODE REVIEW

File: travis/ipa-init.sh [hunk 1]
```diff
@@ -22,7 +22,12 @@
 set -e
 
 # Enable IPA COPR repo
-dnf copr enable -y @freeipa/freeipa-4-7
+# NOTE: IPA does not build for F30
```
Comment (cipherboy) [inline travis/ipa-init.sh]: Not sure how this correlates to the below, but they should definitely enable this. Do you mean `F31`?

Comment (SilleBille) [reply travis/ipa-init.sh]: The `freeipa-master-nightly` [COPR project](https://copr.fedorainfracloud.org/coprs/g/freeipa/freeipa-master-nightly/) does not include F30. So, you will see an error if you try to enable this repo in F30

File: travis/ipa-init.sh [hunk 2]
```diff
@@ -22,7 +22,12 @@
...
+. /etc/os-release
+if [[ `echo "$VERSION_ID"` -ge 31 ]]
```
Comment (cipherboy) [inline travis/ipa-init.sh]: So should this line be `30` then instead of `31`?

Comment (SilleBille) [reply travis/ipa-init.sh]: I just worked with IPA to enable it for F30. I will be removing this condition :-)

To answer your question, `-ge` stands for greater than or equals. So, F31 is right :-)

Comment (cipherboy) [reply travis/ipa-init.sh]: Now it is right, but when we first discussed this, your comment said `F30` didn't work but your check was saying `F30` worked... ;-)

**Pull request merged**